tests/Unit/Dto/PackageTest.php

```diff
@@ -32,21 +32,21 @@ class PackageTest extends TestCase
     /**
      * @inheritdoc
      */
-    public function setUp()/* The :void return type declaration that should be here would cause a BC issue */
```

Comment:
IIRC the comment was copieed from the parent method. I'm not sure what it means though and if it will affect our component.

Reply:
This comment was originally added to preserve backwards compatibility with PHP < 7.1. It was dropped with PHPUnit 7 and replaced by a `: void` return (see https://github.com/sebastianbergmann/phpunit/issues/3288).

Since we do not support PHP < 7.1 and require at least PHPUnit 7.5.x, the `setUp` method signature was streamlined with the base classes.

Reply:
👍